core/src/test/scala/com/intenthq/pucket/TestLogging.scala

```diff
@@ -9,7 +9,6 @@ trait TestLogging {
   SLF4JBridgeHandler.removeHandlersForRootLogger()
   LogManager.getLogManager.reset()
   SLF4JBridgeHandler.install()
-  Logger.getLogger("global").setLevel(Level.OFF)
```

Comment:
`specs2` now uses logging properly and this line now hides `specs2` itself - not necessary to have anyway.